README.md

```diff
@@ -1,30 +1,37 @@
-# DirectAI
+# [DirectAI](https://calendly.com/directai/demo)
```

Comment:
this might look kind of ugly. any strong opinions? I can shift it to another location if you think it'd be better.

Reply:
I agree it's kind of ugly .. already linking the Calendly at the end, I think it's fine to leave it out here

Reply:
maybe replace it with a link to the website? I notice we're not linking that anywhere ?